grumpy-tools-src/grumpy_tools/compiler/block.py

```diff
@@ -364,16 +370,40 @@ def __init__(self, node):
     BlockVisitor.__init__(self)
     self.is_generator = False
     node_args = node.args
-    args = [a.arg for a in node_args.args]
+    args = []
+    for arg in node_args.args:
+      if isinstance(arg, ast.Tuple):
+        args.append(arg.elts)
+      else:
+        args.append(arg.arg)
+     # args = [a.arg for a in node_args.args]
+
```

Comment:
args = [arg.elts if isinstance(arg, ast.Tuple) arg.arg for arg in node_args.args]

Reply:
```python
args = [(arg.elts if isinstance(arg, ast.Tuple) else arg.arg) for arg in node_args.args]
```

I am wondering if this version is more readable than the original

Reply:
It's just the pythonic way, but you'r right . the original is more readable.